roles/routeros_common/molecule/default/create.yml

```diff
@@ -3,21 +3,40 @@
   hosts: localhost
   connection: local
   gather_facts: false
-  # no_log: "{{ molecule_no_log }}"
+  no_log: "{{ molecule_no_log }}"
   tasks:
     # TODO: Developer must implement and populate 'server' variable
 
+    - name: Ensure Vagrant instances are up
+      ansible.builtin.command:
+        cmd: "vagrant up {{ item.name }}"
+      throttle: 1
+      loop: "{{ molecule_yml.platforms }}"
+
+    - name: Get Ansible test inventory
+      ansible.builtin.command:
+        cmd: "../../../../inventories/test/routeros_hosts.py"
+      register: inventory
+      changed_when: false
+
+    - name: Transform inventory into Molecule compatible format
+      ansible.builtin.set_fact:
+        server: {
+          'changed': true,
+          'results': "{{ (inventory.stdout | from_json)._meta.hostvars }}",
```

Comment:
is this where you're losing `common_vars` that causes the issue of not having `ansible_user` and `ansible_port` above? (notice you're grabbing just hostvars)

Reply:
No longer broken.